shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -2455,7 +2467,10 @@ private void updateRecursively(
         }
       }
 
+      int previousNodeId = -1;
       for (SemanticsNode child : childrenInTraversalOrder) {
+        child.previousNodeId = previousNodeId;
```

Comment:
It's guaranteed that we will run through this code again when the order changes and with that a child's previousNodeId changes?

Reply:
yes, this method will called whenever the bridge receive update from framework.